typed-racket-test/optimizer/run.rkt

```diff
@@ -35,7 +35,9 @@
           ;; making program output identical on RacketCS is often worthwhile,
           ;; but optimization logs are too fragile in some cases
           (check-equal? (list (set-subtract log expected-log) (set-subtract expected-log log)) (list (list) (list))))
-        (check-equal? (regexp-split "\n" output) (regexp-split "\n" expected-output))))))
```

Comment:
I would do 
```
(let ([linebreak (if (eq? (system-type 'os) 'windows) "\r\n" "\n"])
  (check-equal? (regexp-split linebreak output) (regexp-split linebreak expected-output))))))
```

Reply:
But I am wondering if there is a better way to get the platform-specific linebreak in Racket.

Reply:
I tried something similar with "\r?\n"
However, this did not work. The last element still had an "\r". I'm guessing that the way `expected-output` is constructed, the last \n gets stripped.

Reply:
I think we should just fix the file and then not make this change. Our files should just be expected to have Unix line endings.

Reply:
I used port->lines

Reply:
If the tests are reading local checkout files, some git configuration will automatically convert Unix line breaks to Windows line breaks.

The xml tests had that issue before 
https://github.com/racket/racket/blob/master/pkgs/racket-test/tests/xml/test.rkt#L91

If #:mode is 'text, \r\n should've been converted to \n though.

Reply:
Thanks @shhyou, that was the right solution.